packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -153,4 +171,21 @@ export class OfflineQueueLink extends ApolloLink {
       });
     }
   }
+
+  private updateIds(
+    operationEntry: OperationQueueEntry,
+    result: FetchResult
+  ): void {
+    const operation = operationEntry.operation;
+    const optimisticResponse = operationEntry.optimisticResponse;
+    if (optimisticResponse && optimisticResponse[operation.operationName].__optimisticId) {
+      const optimisticId = optimisticResponse[operation.operationName].id;
+      this.opQueue.forEach(({ operation: op }) => {
+        if (op.variables.id === optimisticId && result.data) {
+          op.variables.id = result.data[operation.operationName].id;
+        }
+      });
+      this.storage.setItem(this.key, JSON.stringify(this.opQueue));
```

Comment:
This is duplicate (we do that already outside this method)

Reply:
ok, I overlooked this, will fix it